translations.txt

```diff
@@ -390,3 +390,232 @@ This will reboot the system=
+No Keyfile=
+No USB flash configuration data=
+There is a problem with your USB Flash device=
+No Flash=
+Cannot access your USB Flash boot device=
+There is a physical problem accessing your USB Flash boot device=
+Multiple License Keys Present=
+There are multiple license key files present on your USB flash device and none of them correspond to the USB Flash boot device Please remove all key files, except the one you want to replace, from the /boot/config directory on your USB Flash boot device Alternately you may purchase a license key for this USB flash device If you want to replace one of your license keys with a new key bound to this USB Flash device, please first remove all other key files first=There are multiple license key files present on your USB flash device and none of them correspond to the USB Flash boot device. Please remove all key files, except the one you want to replace, from the **/boot/config** directory on your USB Flash boot device. Alternately you may purchase a license key for this USB flash device. If you want to replace one of your license keys with a new key bound to this USB Flash device, please first remove all other key files first.
+BLACKLISTED=
+Blacklisted USB Flash GUID=
+This USB Flash boot device has been blacklisted This can occur as a result of transferring your license key to a replacement USB Flash device, and you are currently booted from your old USB Flash device A USB Flash device may also be blacklisted if we discover the serial number is not unique – this is common with USB card readers=This USB Flash boot device has been blacklisted. This can occur as a result of transferring your license key to a replacement USB Flash device, and you are currently booted from your old USB Flash device. A USB Flash device may also be blacklisted if we discover the serial number is not unique – this is common with USB card readers
+USB Flash device error=
+This USB Flash device has an invalid GUID Please try a different USB Flash device=This USB Flash device has an invalid GUID. Please try a different USB Flash device
+USB Flash has no serial number=
+Trial Requires Internet Connection=
+Cannot validate Unraid Trial key=
+Your Trial key requires an internet connection Please check Settings > Network=Your Trial key requires an internet connection. Please check Settings > Network
+Stale=
+Stale Server=
+Please refresh the page to ensure you load your latest configuration=
+Registration Wizard=
+To Registration Wizard Home=
+Continue Trial=
+Continue to Unraid=
+Toggle server info visibility=
+You can safely close this window=
+Automatically closing in=
+bye, bye 👋=
```

Comment:
Untested with extended characters

Reply:
It seems to work fine when I upload ZIPs with translations for German, French, & Spanish. But I will re-confirm.

Reply:
A ton easier IMO to simply add the appropriate line to say nl_NL with a sample translation and the delete the applicable .dot file.

Reply:
Confirming that it works with an emoji in the key.

![Screen Shot 2020-12-08 at 09 08 08](https://user-images.githubusercontent.com/2531682/101516956-ff586d80-3934-11eb-9586-4d0b890fa03b.png)